tests/functional/misc.sh

```diff
@@ -13,6 +13,9 @@ source common.sh
 # Can we ask for the version number?
 nix-env --version | grep "$version"
 
+nix_env=$(type -P nix-env)
+(PATH=""; ! $nix_env --help 2>&1 ) | grepQuiet -F "The 'man' command was not found, but it is needed for 'nix-env' and some other 'nix-*' commands' help text. Perhaps you could install the 'man' command?"
+
 # Usage errors.
 expect 1 nix-env --foo 2>&1 | grep "no operation"
 expect 1 nix-env -q --foo 2>&1 | grep "unknown flag"
```

Comment:
Should we be using `expectStderr`?

Reply:
Normally I would, but I've also cleared `PATH` here, so wasn't sure.
It's fine.